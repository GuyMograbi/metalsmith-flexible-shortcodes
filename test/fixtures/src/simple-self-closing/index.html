---
title: HTML
---
<p>
This shows a simple self enclosing shortcode in HTML: [makro]
</p>