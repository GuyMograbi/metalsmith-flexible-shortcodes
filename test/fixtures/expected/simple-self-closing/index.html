<p>
This shows a simple self enclosing shortcode in HTML: I'm replaced :)
</p>